src/docs/feature-flags.mdx

```diff
@@ -68,7 +68,7 @@ add a third optional boolean parameter when adding the feature, for example:
 default_manager.add('organizations:test-feature', OrganizationFeature, True)
 ```
 
-If you don't plan to use flagr don't pass this third parameter, for example: 
+If you don't plan to use flagr don't pass this third parameter, for example:
```

Comment:
Could we expand this or the previous paragraph to explain when not to use flagr (could take it from the notion) and to use options instead, then link to the new section?

Reply:
Done! Added a note below